### PR TITLE
Docs: Link to the new alerting API spec

### PR DESCRIPTION
**Pull request by papagian**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/http_api/alerting.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ aliases = ["/docs/grafana/latest/http_api/alerting/"]
 
 # Alerting API
 
+>**Note:** This information is for the [legacy dashboard alerts]({{< relref "../alerting/old-alerting/_index.md" >}}). A preview of Grafana 8 Alerts API specification can be found [here](https://editor.swagger.io/?url=https://raw.githubusercontent.com/grafana/grafana/main/pkg/services/ngalert/api/tooling/post.json).
```
Comment (papagian) [inline docs/sources/http_api/alerting.md:10]: This link is for the upstream version of the API. For the v8.0.x (latest) docs version should be:
`https://editor.swagger.io/?url=https://raw.githubusercontent.com/grafana/grafana/v8.0.x/pkg/services/ngalert/api/tooling/post.json` instead.

I wonder if there is a way to detect the target version of the docs (eg from the URI) and replace it appropriately.
Alternatively, we need to maintain this separately for each version which is harder and error prune.

Comment (achatterjee-grafana) [reply docs/sources/http_api/alerting.md:10]: Since the new 8.0 API is an opt in feature, detecting the version is not needed at this time.

Comment (achatterjee-grafana) [inline docs/sources/http_api/alerting.md:10]:
```suggestion
>**Note:** This topic is relevant for the [legacy dashboard alerts]({{< relref "../alerting/old-alerting/_index.md" >}}) only. 

You can find Grafana 8 alerts API specification details [here](https://editor.swagger.io/?url=https://raw.githubusercontent.com/grafana/grafana/main/pkg/services/ngalert/api/tooling/post.json).  Also, refer to [Grafana 8 alerts documentation]({{< relref "../alerting/unified-alerting/_index.md" >}}) for details on how to create and manage new alerts.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added my thoughts.

File: docs/sources/http_api/alerting.md [hunk 2]
```diff
@@ -7,7 +7,9 @@ aliases = ["/docs/grafana/latest/http_api/alerting/"]
 
 # Alerting API
 
->**Note:** This information is for the [legacy dashboard alerts]({{< relref "../alerting/old-alerting/_index.md" >}}). A preview of Grafana 8 Alerts API specification can be found [here](https://editor.swagger.io/?url=https://raw.githubusercontent.com/grafana/grafana/main/pkg/services/ngalert/api/tooling/post.json).
+>**Note:** This topic is relevant for the [legacy dashboard alerts]({{< relref "../alerting/old-alerting/_index.md" >}}) only. 
+
+You can find Grafana 8 alerts API specification details [here](https://editor.swagger.io/?url=https://raw.githubusercontent.com/grafana/grafana/main/pkg/services/ngalert/api/tooling/post.json).  Also, refer to [Grafana 8 alerts documentation]({{< relref "../alerting/unified-alerting/_index.md" >}}) for details on how to create and manage new alerts.
 
 You can use the Alerting API to get information about alerts and their states but this API cannot be used to modify the alert.
 To create new alerts or modify them you need to update the dashboard json that contains the alerts.
```
Comment (achatterjee-grafana) [inline docs/sources/http_api/alerting.md:15]:
```suggestion
You can use the Alerting API to get information about legacy dashboard alerts and their states but this API cannot be used to modify the alert.
To create new alerts or modify them you need to update the dashboard JSON that contains the alerts.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Just one more change.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM!

**Pull request merged**